tests/e2e/tests/regression/browser/add-keys.e2e.ts

```diff
@@ -44,3 +53,55 @@ test('Verify that user can create different types(string, number, null, array, b
         }
     }
 });
+// https://redislabs.atlassian.net/browse/RI-3995
+test
+    .before(async() => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig, ossStandaloneConfig.databaseName);
```

Comment:
you need to user ossStandaloneBigConfig here
because the main point of this feature is to display the new key at the top in huge databases where previously keys were added somewhere at the middle

Reply:
and seems like with big database you will be able to add test 'Verify that user can see the key removed from the top when refresh List view'